panc/src/main/java/org/quattor/pan/dml/data/ProtectedListResource.java

```diff
@@ -25,7 +25,11 @@ public Element duplicate() {
 
 	@Override
 	public Element get(Term key) throws InvalidTermException {
-		return baseList.get(key);
+		final Element value = baseList.get(key);
```

Comment:
@gombasg is this really needed? any element in a protected resource should be protected already. (or is there a way to make a protectedresource with unprotected elements in it? or is the way to enforce that shallow copy of a protected resource becomes a deep-protected copy.